debezium-core/src/main/java/io/debezium/data/HStore.java

```diff
@@ -0,0 +1,21 @@
+package io.debezium.data;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+
+//TODO-> Add Javadocs
+public class HStore {
+
+    public static final String LOGICAL_NAME="io.debezium.data.hstore";
+
+    public static SchemaBuilder builder(){
+        return SchemaBuilder.map(SchemaBuilder.string().name("key"),
```

Comment:
I think the Kafka Connect Data API actually supports map types, so we probably can and should use that one instead of adding a custom type for it. WDYT?

Reply:
If its there we should use it rather then reinventing. I will have a look at it

Reply:
To what I understand , my approach will be to remove HStore Data Type and in PostGresValueConvertor.java , I will just add SchemaBuilder.map(SchemaBuilder.string().name("key"),SchemaBuilder.string().name("value")) under MAP case .